.drone.yml

```diff
@@ -625,7 +625,102 @@ volumes:
     temp: {}
 
 ---
+kind: pipeline
+type: kubernetes
+name: tag-stage-terraform-provider
+
+trigger:
+  event:
+    - tag
+  ref:
+    include:
+      - refs/tags/terraform-provider-teleport-v*
+
+depends_on:
+  - tag-build-terraform-linux
+  - tag-build-terraform-darwin
+
+concurrency:
+  limit: 1
+
+steps:
+  - name: Upload terraform provider to staging registry
+    image: golang:1.17.5
```

Comment:
I think, we could parameterise the required Go version as it seems equal for the whole suite (if Drone allows it).